supported-standards/lens-metadata-standards/publication/shared-ts-interfaces/metadata-common.ts

```diff
@@ -0,0 +1,52 @@
+import { AppId, Locale, Markdown } from './type-alias';
+
+enum PublicationContentWarning {
+  NSFW = 'NSFW',
+  SENSITIVE = 'SENSITIVE',
+  SPOILER = 'SPOILER',
+}
+
+export interface MetadataCommon {
+  /**
+   * The metadata lens_id can be anything but if your uploading to IPFS
+   * you will want it to be random.. using uuid could be an option!
+   */
+  metadata_id: string;
+
+  /**
+   * The content of a publication.
+   */
+  content?: Markdown;
```

Comment:
There is an interesting use case emerging from current apps. They `name` field (historically reserved for marketplace NFT name) is used as a title for the publication in these scenarios:
https://github.com/lens-protocol/lens-sdk/issues/482

As with Publication Metadata v3 we are making an explicit effort to segregate marketplace data from Lens specific data, I think the same way `content` is part of the `MetadataCommon` we should promote `title?: string` to `MetadataCommon` too.

Currently the `title?:  string` is only part of some to of the sub-specifications.

Thoughts?